portingdb/htmlreport.py

```diff
@@ -588,6 +589,82 @@ def ascending(p):
     )
 
 
+def mispackaged():
+    # Parameters
+    requested = request.args.get('requested', None)
+    if requested not in ('1', None):
+        abort(400)  # Bad request
+
+    # Database
+    db = current_app.config['DB']()
+    query = queries.packages(db)
+
+    # Get data
+    mispackaged, query = queries.split(query, tables.Package.status == 'mispackaged')
+    mispackaged = mispackaged.options(subqueryload('collection_packages'))
+    mispackaged = mispackaged.options(subqueryload('collection_packages.links'))
+    mispackaged = mispackaged.options(subqueryload('collection_packages.tracking_bugs'))
+
+    if requested:
+        # Filter only to packages where maintainer requested a patch
+        mispackaged = [pkg for pkg in mispackaged
+                if "https://bugzilla.redhat.com/show_bug.cgi?id=1333765"
+                in pkg.list_tracking_bugs]
+
+    # Sorting the packages based on their last_link_update
+    #   Note that sorting through the database is problematic, because
+    #   to sort on a @property you need to create a @hybrid_property which
+    #   needs to be written as sort of an SQL expression
```

Comment:
Well, it's a bit problematic, because some packages have more Link-bug fields, some of which are Null on the last_update and some of which are not, and some packages just have a Link-bug with a Null on last_update. I'll remove the comment if it doesn't matter if I sort it in Python anyway.